config/environments/test.rb

```diff
@@ -0,0 +1,64 @@
+require "active_support/core_ext/integer/time"
```

Comment:
- [ ]  You are using Swagger, but the "test" folder still exists, which is used by another test framework. Kindly consider deleting that folder if you are not using it. 👍